app/views/dashboards/show.html.erb

```diff
@@ -13,30 +13,30 @@
     </aside>
 
     <div class="crayons-layout__content">
-      <% if @articles.any? %>
-        <div class="flex items-center justify-between mb-3">
-          <h2 class="hidden m:block fs-l fw-heavy">Posts</h2>
-
-          <div class="flex flex-wrap">
-            <% if params[:which] != "organization" %>
-              <% if @articles.any? {|article| article.archived } %>
-                <%= link_to "Show archived", "javascript:;", id: "toggleArchivedLink", class: "crayons-btn crayons-btn--outlined ml-2 whitespace-nowrap my-1" %>
-              <% end %>
+      <div class="flex items-center justify-between mb-3">
+        <h2 class="hidden m:block fs-l fw-heavy">Posts</h2>
 
-              <%= select_tag "dashboard_sort", options_for_select(sort_options, params[:sort]), 'aria-label': "Sort By", class: "crayons-select w-auto ml-2 my-1" %>
+        <div class="flex flex-wrap">
+          <% if params[:which] != "organization" %>
+            <% if @articles.any?(:archived) %>
```

Comment:
I can see you have changed this IF statement but now the button doesn't appear at all for me (even though I do have archived posts)

Reply:
Ugh, typo, it's missing a `&`.

Reply:
This has been fixed.